frontend/console/e2e/cron.spec.ts

```diff
@@ -24,3 +24,33 @@ test('send cron request', async ({ page }) => {
 
   expect(responseJson).toEqual({})
 })
+
+test('submit cron form using ⌥+⏎ shortcut', async ({ page }) => {
+  await navigateToDecl(page, 'cron', 'thirtySeconds')
+
+  await page.locator('input#request-path').focus()
+
+  await page.keyboard.press('Alt+Enter')
+
+  const responseEditor = page.locator('#response-editor .cm-content[role="textbox"]')
+  await expect(responseEditor).toBeVisible()
+
+  const responseText = await responseEditor.textContent()
+  const responseJson = JSON.parse(responseText?.trim() || '{}')
+
+  expect(responseJson).toEqual({})
+})
+
+test('submit cron form using ⌥+⏎ shortcut without focusing first', async ({ page }) => {
+  await navigateToDecl(page, 'cron', 'thirtySeconds')
+
+  await page.keyboard.press('Alt+Enter')
```

Comment:
I'm surprised we have to use the `alt` key here, but maybe that's just how this works :)

Reply:
I think this is because of the difference between using document and window. Switching to window will allow the command key. Will update.